dependencies.py

```diff
@@ -0,0 +1,55 @@
+from database import SessionLocal
+from fastapi import FastAPI, Depends
+from typing import Annotated
+import httpx
+from sqlalchemy.orm import Session
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+def common_city_item_params(db: Session, city_id: int) -> dict:
```

Comment:
The function 'common_city_item_params' is defined to take a 'Session' and 'city_id' as parameters, but it does not use FastAPI's dependency injection. Consider using 'Depends' to inject the 'Session' if this function is intended to be used as a dependency.